client/lib/cart/store/index.js

```diff
@@ -98,16 +99,8 @@ function update( changeFunction ) {
 	cartAnalytics.recordEvents( previousCart, nextCart );
 }
 
-CartStore.dispatchToken = Dispatcher.register( function( payload ) {
-	var action = payload.action,
-		cartItem;
-
-	if ( action.cartItem ) {
-		cartItem = cartValues.fillInSingleCartItemAttributes(
```

Comment:
`fillInSingleCartItemAttributes` is invoked in `update` method through `fillInAllCartItemAttributes` method :)
There is no need to call it twice.